tsconfig.json

```diff
@@ -0,0 +1,106 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
Do we want to keep all of the commented code?

Reply:
This file is generated so i'd just keep it as is.